app/assets/stylesheets/onboarding.scss

```diff
@@ -0,0 +1,66 @@
+@import 'variables';
+@import 'mixins';
+@import 'config/import';
+
+.onboarding-task-card {
+  background: #3b49df;
```

Comment:
are there any global colors that already work for this? if there is, we should use that instead (i suspect there is?).

if not, let's use a `$` prefixed variable at the top of this class, even if it's just in the context of this one file.

Reply:
Oops left this out. Thanks!